server/study_groups/methods.js

```diff
@@ -454,3 +454,59 @@ Meteor.methods({
     return true;
   }
 });
+
+Meteor.methods({
+  transferStudyGroup: function(data) {
+    check(data, {
+      studyGroupId: String,
+      newOwnerId: String,
+      studyGroupTitle: String,
+      studyGroupSlug: String
+    });
+
+    if (!this.userId) {
+      throw new Meteor.Error('Members.methods.editStatus.not-logged-in', 'Must be logged in to transfer study group.');
+    }
+
+    const actor = Meteor.user()
+    if (!actor || !Roles.userIsInRole(actor, ['owner','admin', 'moderator', 'member'], data.studyGroupId )) {
+      throw new Meteor.Error(403, "Access denied");
+    }
+
+    //Remove old owner and update role of new owner to 'owner'
+    StudyGroups.update({_id: data.studyGroupId, 'members.id': actor._id },
+                     {$set: {'members.$.role': 'member'} });
+    StudyGroups.update({_id: data.studyGroupId, 'members.id': data.newOwnerId },
+                     {$set: {'members.$.role': 'owner'} });
+    Roles.setUserRoles(actor._id, 'member', data.studyGroupId);
```

Comment:
Instead of demoting current `owner` to a `member` role, what if we just make them an `admin` ?